Main call IRC Minutes from 15.10.2025 (#1244)
w3c/wot: minutes/main/20251015/Readme.md

# IRC Minutes from 15.10.2025

Attendees

Present
    benfrancis
    
Regrets
    -
Chair
    sebastian
Scribe
    Mahda

Contents

    Minutes
    Calander Events
    Publication Process
    Charter

## Meeting minutes

> the transcription from AI was fnot bad but sometimes special terms were not correct, we need to go through it

Reviewer: ```suggestion
the transcription from AI was not bad but sometimes special terms were not correct, we need to go through it
```

## Minutes

sk: AI generated minutes from last week

> … the transcription from AI was fnot bad but sometimes special terms were not correct, we need to go through it

Reviewer: ```suggestion
… the transcription from AI was not bad but sometimes special terms were not correct, we need to go through it
```

> ege: from other calls e.g, TD call, what should be the process? should the minutes be approved by the moderators?

Reviewer: ```suggestion
ege: from other calls, e.g., TD call, what should be the process? should the minutes be approved by the moderators?
```

Dave: It may start by itself, we should check

daniel: I started the call today with scripting, Zoltan and I were the only one present, did it record anything? Do I need to perform something?

Dave: I haven't seen anything till now, got to check after the event

sk: we need to evaluate this again

sk: the IRC minutes to be scanned and corrected. I put the AI minutes in the WoT repository, a folder that is called Minutes, you could do a similar process for TD

ege: should the moderator do the pull request, I don't know how to get the content of the minutes

sk: the TD has its own repo, it is important that you provide a link then. We have alot of WoT-based calls like marketing, I am fine to record the minutes in their own space

ege: How do we get the minutes?

sk: I understand thet the host is the one receiving it, and needs to be forwarded

Dave: can anyone open the AI companion in zoom?

Ege: I think until this is sorted our we need to use IRC, it puts more load on the moderator

Dave: I think in the chairs call on Monday, we discussed that we should rely on the IRC and the AI companion is additional

> sk: we are on an experimental mode, but if it turns out that we have much more work we should go back to the old procedure with IRC

Reviewer: ```suggestion
sk: we are in an experimental mode, but if it turns out that we have much more work, we should go back to the old procedure with IRC
```

would be to focus on the WoT TD. Of course the WoT TD cannot be a technology and a task that can exist alone, you need inputs like the bindings and the registry, but topics like discovery, profiles and architecture I am not sure whether we have the men power for it.
… I am open to discuss this in the new charter, but if I feel we don't have the men power we focus on the TD 2.0 and then to make another charter related to the other aspects. I have the feeling that other working groups do a similar approach focusing on smaller topics.
… I would like to hear your views on this aspect

Ben: I completly agree that the TD is the core and the number of meetings and the calls we had in the past were not sustainable. The question is which one survives? The problem I have to only focusing on TD and registry is that we never use the vocabulary from HTTP bindings, to adopt bindings we need a better way to figure out how they will work

> with the Profiles. I think if we have to wait for 2 years to then start working on Profiles, this will be too long. I would like to see WoT profiles in the next charter. I do see we need to drop Architecture, dicovery, and security merged with other specs.

Reviewer: ```suggestion
Ben: I completly agree that the TD is the core and the number of meetings and the calls we had in the past were not sustainable. The question is, which one survives? The problem I have with only focusing on TD and registry is that we never use the vocabulary from HTTP bindings; to adopt bindings, we need a better way to figure out how they will work with the Profiles. I think if we have to wait for 2 years to start working on Profiles, this will be too long. I would like to see WoT profiles in the next charter. I do see we need to drop Architecture, dicovery, and security merged with other specs.
```
